### PR TITLE
Added email signup test

### PR DESCRIPTION
**Pull request by chen-robert**
Fixes part of https://github.com/publiclab/plots2/issues/6303

I added a test for the email signup route

### CODE REVIEW

File: test/integration/signup_flow_test.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      spamaway: {
+        statement1: I18n.t('spamaway.human.statement1'),
+        statement2: I18n.t('spamaway.human.statement1'),
+        statement3: I18n.t('spamaway.human.statement1'),
+        statement4: I18n.t('spamaway.human.statement1')
+      }
+    }
+
+    assert response.body.include? '1 errors prohibited this user from being saved'
+    assert response.body.include? 'Email should look like an email address.'
```
Comment (SidharthBansal) [inline test/integration/signup_flow_test.rb]: 'Email should look like an email address.' fto needed here too

File: test/integration/signup_flow_test.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class SignUpTest < ActionDispatch::IntegrationTest
+  def setup
+    @new_user = {
+      :username => "newuser",
+      :email => "newuser@gmail.com",
+      :password => "validpassword"
+    }
+  end
+
+  test 'email error messages' do
```
Comment (SidharthBansal) [inline test/integration/signup_flow_test.rb]: Grammar

**Comment by SidharthBansal** [comment]
Kindly rerun Travis for all prs which you did

**Comment by chen-robert** [comment]
I commented out a test that will be fixed when https://github.com/publiclab/plots2/pull/6965 is merged.

**Comment by newbazz** [comment]
you have 16 commits for one PR, ca you please squash them into one.

**Comment by chen-robert** [comment]
@newbazz You should be able to squash the commits when you merge the PR

**Comment by newbazz** [comment]
First of all, i am not the maintainer and second of all having one commit for one pr is a good practice.

**Comment by chen-robert** [comment]
@newbazz thanks for the advice, I've squashed it all down now :rocket:

**Comment by jywarren** [comment]
Love this!!!! Thank you!!!

**Pull request merged**